en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,77 @@
+
+- `compile_nuttx.yml`: Builds NuttX flight controller board targets. It can detect Flash overflows and code syntax errors
+- `sitl_tests.yml`: Executes [MAVSDK SITL Tests](./integration_testing_mavsdk.md). It can detect functional bugs in the code (e.g. VTOL doesn't return to home)
+- `mavros_mission_tests.yml`: Executes Mission plans using MAVROS. It can check for functional compatibility between MAVROS and PX4
+- `checks.yml`: Runs various make targets (using `make` command). The commands for each targets are documented in the [Makefile](https://github.com/PX4/PX4-Autopilot/blob/main/Makefile)
+  - `check_format`: Checks the coding style format using Clang Tidy ?
+  - `airframe_metadata`: Builds airframe XML metadata
+  - `parameters_metadata`: Builds parameter XML metadata
+  - `module_documentation`: Builds module documentation
+  - `tests`: Runs unit tests
+  - `tests_coverage`: Runs coverage tests
+  - ... (other auxiliary tests)
+- `ekf_functional_change_indicator.yml`: Check if there's any functionality change in EKF ?
+- `ekf_update_change_indicator.yml`: Check if there's any functionality change in EKF ?
+- `failsafe_sim.yml`: Builds [failsafe simulator](../config/safety_simulation.md)
+- `clang-tidy.yml`: Runs Clang Tidy to check if coding style is consistent
+- `compile_linux.yml`: Builds linux flight controller board targets
+
+## PX4 release / main branch artifacts (+ Board specific files)
+Upon changes to the `main` branch, or to `release/*` branches (any releases), certain artifact files need to be built and uploaded to the server.
+This is handled by [this](https://github.com/PX4/PX4-Autopilot/blob/main/.github/workflows/deploy_all.yml) workflow.
+
+For each board targets, the artifacts built are:
+- Parameter metadata (`parameters.*`)
+- Events metadata (`events/*.xz`)
+- Actuators metadata (`actuators.json*`)
+- Target binary (`*.px4`)
+
+Then these files get uploaded to the [Amazon S3 bucket server](https://px4-travis.s3.amazonaws.com/).
+
+### Usage by QGC
+These files then get used in QGroundControl for:
+- Flashing PX4 binaries (stable: `release/*` / beta: `main`) through Firmware tab
+- Downloading & Caching the Events / Actuators / Parameters metadata for boards that doesn't support [MAVLink Component Information service](https://mavlink.io/en/services/component_information.html) based metadata retrieval
+
+Without these artifacts getting uploaded, QGroundControl would quickly go out of sync / won't be able to download & flash target binaries. So this is a crucial part of CI.
+
+## Generic metadata for Documentation & QGC by main branch (Not board specific)
+Whenever there are changes to the `main` branch, separately from the `deploy_all.yml` workflow above, Jenkins builds / deploys metadata.
+Jenkins file can be found [here](https://github.com/PX4/PX4-Autopilot/blob/main/Jenkinsfile).
```

Comment:
FYI generally better to provide the name of the file path rather than "here". Makes it easier for users to find who are working on their local source doing updates.